docs/source/api_references.rst

```diff
@@ -27,11 +27,17 @@ Core API
 .. autosummary::
     :toctree: api
     :nosignatures:
+    :template: classtemplate.rst
 
-    datamodule
-    decorators
-    hooks
-    lightning
+    ~hooks.CheckpointHooks
```

Comment:
it makes it so that only the last part of the dotted path shows in the rendered page. If we write
`~x.y.z.CheckpointHooks` it will only show as CheckpointHooks